### ISSUE TITLE
FilePipe writeLineSeparator writes newline in separate write action allowing other threads to write between output and newline 

### ISSUE THREAD

**Issue by jjansenvr** [ibissource/iaf#577]
**Describe the bug**
Using writeLineSeparator in a multithreading adapter gave a invalid result
**result:** 
```
line1<Newline>
line2line3<Newline>
<Newline>
line4<Newline>
```

**expected result**
```
line1<Newline>
line2<Newline>
line3<Newline>
line4<Newline>
```

**Category**
 - Frank Framework

**To Reproduce**
- create an adapter writing in bulk to a file using multiple threads
- apply load

**Expected behavior**
By adding the newline to the string to write and write in 1 go this problem will no longer occur 

**Environment:**
 - websphere
 - IAF 7.5-20200124.165603

**Additional context**
Add any other context about the problem here. (f.e. ladybug report / test adapter with larva test)

**Comment by gvanbrakel** [comment]
Although we could try to send the line separator in some way 'closer' to the body of the file, there is and will be no guarantuee that the files sent from multiple threads will be in strictly consecutive order. I think the correct pattern would be make the multiple threads each write their files to a common directory, and have a single thread combine those files into a single file, via a DirectoryListener.

**Comment by jjansenvr** [comment]
Hi Gerrit

I think doing a single write action with the newline added to the string before writing it to file will solve this issue. I understand the order of the rows cannot be guaranteed. 
The option of writing to different files and merging them will still not guaranty order only solve this issue and be quite complex. 

please try to add the newline in after the string and then write to file

**Comment by jjansenvr** [comment]
@gvanbrakel Was this fix pickedup? as I remember it it was just adding the EOL to the string before writing it

**Comment by alisihab** [comment]
Possible fix will be applied with the PR #1290.
